libs/domain/cart/services/src/default-cart.service.ts

```diff
@@ -291,7 +291,7 @@ export class DefaultCartService implements CartService {
     return this.executeWithOptionalCart(qualifier, this.updateCartCommand$);
   }
 
-  setActiveCart(qualifier: UpdateCartQualifier): Observable<unknown> {
+  setDefaultCart(qualifier: UpdateCartQualifier): Observable<unknown> {
```

Comment:
do we need this method as the qualifier for update already allows to do this?

Reply:
I was thinking about it, and made a conclusion to make an additional method for this. But now, I'm not sure. Lets drop it